src/app/my-info/(account)/reviews/_components/MyReviewProduct.tsx

```diff
@@ -1,52 +1,131 @@
 'use client';
 
+import { deleteUserProductReview } from '@/api/productReviewAPI';
+import { Dialog, Modal } from '@/components';
+import WriteEditModal from '@/components/WriteEditModal/WriteEditModal';
+
 import { formatDateToString } from '@/libs/formatDateToString';
-import { ProductType } from '@/types/ProductTypes';
+
+import type { ReviewDto } from '@/types/ProductReviewTypes';
+
+import { QueryClient, useMutation } from '@tanstack/react-query';
 import classNames from 'classnames/bind';
 import Image from 'next/image';
 import Link from 'next/link';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { ProductType } from '@/types/ProductTypes';
```

Comment:
```suggestion
import type { ProductType } from '@/types/ProductTypes';
```

요부분은 항상 내가 작성할 땐 안보이는데 남의 코드 볼 때는 잘 보이는 마법,,